MechJeb2/MechJebModuleThrustController.cs

```diff
@@ -890,6 +871,52 @@ static void MaxThrust(double[] x, ref double func, double[] grad, object obj)
             }
         }
 
+        /// <summary>
+        /// Handles auto-RCS ullaging up to very stable.
+        /// </summary>
+        /// <param name="s"></param>
+        private void ProcessUllage(FlightCtrlState s)
+        {
+            if (!autoRCSUllaging || s.mainThrottle <= 0F || throttleLimit <= 0F)
+                return;
+
+            bool isStable = vesselState.lowestUllage == VesselState.UllageState.VeryStable;
+            if (!isStable)
+            {
+                ullageUntil = vesselState.time + 0.15;
+            }
+
+            if (vesselState.time >= ullageUntil) 
+                return;
+
+            // Continue ullaging until at least one engine has spooled up enough to produce >1% of it's rated thrust
+            if (isStable && vesselState.enginesWrappers.Where(e => e.engine.requestedThrottle > 0.01)
+                                                       .All(e => e.engine.currentThrottle < 0.01))
+            {
+                ullageUntil = vesselState.time + 0.15;
+            }
```

Comment:
How about ullaging up until the engines hit 0.95 or something like that?  Be simpler to write and understand.

Reply:
The issue with that would be that nuclear engines take another 20 seconds until the throttle goes from 1% to 95+%. That 1% figure is what RF itself uses for determining whether the engine is in pre-ignition spoolup state. After 1% is reached then the thrust will start increasing very rapidly.
Relevant code for reference: https://github.com/NathanKell/ModularFuelSystem/blob/master/Source/Engines/ModuleEnginesRF.cs#L368-L374